### PR TITLE
Minor changes.

### PR DESCRIPTION
**Pull request by ironmaniiith**

### CODE REVIEW

File: is.js [hunk 1]
```diff
@@ -169,7 +169,7 @@
     };
 
     // is a given value undefined?
-    is.undefined = function(value) {
+    is['undefined'] = function(value) {
         return value === void 0;
```
Comment (jdalton) [inline is.js]: IE8 doesn't have a problem with properties named `'undefined'`. I'd like to avoid superstitious code changes.

Comment (ironmaniiith) [inline is.js]: Ok, I'll remove it. :+1:

**Comment by jdalton** [comment]
Thanks!

**Comment by ironmaniiith** [comment]
I had a question..
What are your suggestions on allowing more then two arguments to be given to is.sameType.?

**Comment by jdalton** [comment]
Seems like something that fits composition, maybe through `Array#reduce`.

**Comment by ironmaniiith** [comment]
I was trying to figure out whether it can be done simply in one iteration, came up with this.
Considering an array A is passed to is.sameType, the possible cases are.

1. A[0] is Nan, then rest all should be Nan.
2. A[0] is Number, then rest all should be Number `and` not NaN.
3. A[0] is of type 'X' (other than above two), then rest all should be of type 'X'.

Now we can make three functions for the above cases and maybe pass to Array#every or Array#reduce, whatever is best.

Please tell me if this approach is having any flaw.

**Comment by ironmaniiith** [comment]
And case 1 and 2 can be combined in a single case where A[0] is Number and should be handled further for NaN case.

**Pull request merged**